scripts/PHNI-JP/c101203042.lua

```diff
@@ -0,0 +1,98 @@
+--エ二グマスター·バックビット
+local s,id,o=GetID()
+function s.initial_effect(c)
+    --synchro summon
+	aux.AddSynchroProcedure(c,nil,aux.NonTuner(nil),1)
+	c:EnableReviveLimit()
+    --tohand
+	local e1=Effect.CreateEffect(c)
+    e1:SetDescription(aux.Stringid(id,0))
+	e1:SetCategory(CATEGORY_TOHAND)
+	e1:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+	e1:SetCode(EVENT_SPSUMMON_SUCCESS)
+	e1:SetProperty(EFFECT_FLAG_CARD_TARGET+EFFECT_FLAG_DELAY)
+	e1:SetCountLimit(1,id)
+	e1:SetCondition(s.sumcon)
+	e1:SetTarget(s.mvtg)
+	e1:SetOperation(s.mvop)
+	c:RegisterEffect(e1)
+    --mobe(to grave)
+	local e2=Effect.CreateEffect(c)
+    e2:SetDescription(aux.Stringid(id,0))
```

Comment:
Duplicate String reference
```suggestion
    e2:SetDescription(aux.Stringid(id,1))
```